zerver/tests/test_markdown.py

```diff
@@ -751,7 +751,7 @@ def make_link(url: str) -> str:
         media_tweet_html = ('<a href="http://t.co/xo7pAhK6n3">'
                             'http://twitter.com/NEVNBoston/status/421654515616849920/photo/1</a>')
 
-        emoji_in_tweet_html = """Zulip is <span aria-label=\"100\" class="emoji emoji-1f4af" role=\"img\" title="100">:100:</span>% open-source!"""
```

Comment:
Might as well get rid of the other unnecessary escape characters in the tests you touched here as well?